CRM/Civicase/Service/CaseCategoryMenu.php

```diff
@@ -32,7 +34,7 @@ public function createItems($caseTypeCategoryName) {
     );
 
     $params = [
-      'label' => ts($caseTypeCategoryName),
+      'label' => ts($labelForMenu),
```

Comment:
The error was there before this PR, but fyi, this is an incorrect use of the "ts" function: https://docs.civicrm.org/dev/en/latest/translation/#avoid-variables-inside-strings

The label should be translated before being sent in the 'ts' function.

Reply:
Hi Mathieu! thanks for your comment.
Actually, this is not a variable inside a string, it is just a variable. I don't see a risk of missing a translation key by this usage.

Reply:
It evaluates to a string. In other words, it's unlikely that gettext/ts would work in this case. Sometimes it accidentally does, when the string is elsewhere in the code, but otherwise it's not a recommended practice. The string extraction will throw warnings.
That said, I know the PR does not change this behaviour, the bug was there before, I just wanted to raise awareness about translation. :)

Reply:
Hey Mathieu, thanks again for your comments and suggestions.
I am getting the idea of what you are pointing out. Honestly, I am not fully aware of the implementation of the `ts` method, but according to your comments, it would be far from being the key => value association that I imagined.
Thanks for raising awareness on this topic 🙂  I will keep an eye on it in future updates.